client/app/kubernetes/translations/Messages_fr_FR.xml

```diff
@@ -0,0 +1,72 @@
+    <translation id="kube_guides_dashboard">Dashboard Kubernetes</translation>
+
+    <translation id="kube_service">Service</translation>
+    <translation id="kube_service_error">Une erreur s'est produite lors du chargement des informations</translation>
+    <translation id="kube_service_description_information">Vous trouverez ci-dessous les informations de votre service Kubernetes, vous permettant d'y accéder à l'aide de l'</translation>
+    <translation id="kube_service_description_tool">outil kubetcl.</translation>
+    <translation id="kube_service_description_reset">Vous pouvez réinitialiser la configuration de votre cluster Kubernetes à tout moment.</translation>
+    <translation id="kube_service_offer_beta">Service managé Kubernetes Beta (gratuit)</translation>
+
+    <translation id="kube_service_file">Fichier kubeconfig</translation>
+    <translation id="kube_service_file_error">Une erreur est survenue lors du chargement du fichier de configuration</translation>
+    <translation id="kube_service_file_help">Le fichier kubeconfig contient les données permettant à l'outil en ligne de commande kubectl de communiquer avec l'API de votre cluster.</translation>
+    <translation id="kube_service_file_more_information">Plus d'information disponible ici</translation>
+
+    <translation id="kube_service_cluster_information">Informations sur le cluster</translation>
+    <translation id="kube_service_cluster_status">Statut</translation>
+    <translation id="kube_service_cluster_status_READY">Ok</translation>
+    <translation id="kube_service_cluster_status_ERROR">Erreur</translation>
+    <translation id="kube_service_cluster_status_INSTALLING">En cours d'installation</translation>
+    <translation id="kube_service_cluster_status_DELETING">En cours de suppression</translation>
+    <translation id="kube_service_cluster_version">Version de Kubernetes installée</translation>
+    <translation id="kube_service_cluster_region">Région</translation>
+    <translation id="kube_service_cluster_api_url">URL de l'API Kubernetes de votre cluster</translation>
+
+    <translation id="kube_service_billing">Facturation</translation>
+    <translation id="kube_service_billing_plan">Offre</translation>
+    <translation id="kube_service_billing_contact">Nic Billing</translation>
+    <translation id="kube_service_billing_renewal">Renouvellement</translation>
+
+    <translation id="kube_nodes">Noeuds</translation>
+    <translation id="kube_nodes_add">Ajouter un noeud</translation>
+    <translation id="kube_nodes_add_region">Durant la bêta, vos nœuds seront tous dans la région GRA5. Nous permettrons ultérieurement de choisir cette région.</translation>
+    <translation id="kube_nodes_add_success">Le noeud a bien été ajouté</translation>
+    <translation id="kube_nodes_add_flavor_family">Choisir un modèle d'instance</translation>
+    <translation id="kube_nodes_add_flavor_family_balanced">General purpose</translation>
```

Comment:
EN text ? or legit ?

Reply:
This is the given name for french translations